packages/plugins/users-permissions/server/services/user.js

```diff
@@ -126,21 +126,17 @@ module.exports = ({ strapi }) => ({
       .get({ key: 'email' })
       .then(storeEmail => storeEmail['email_confirmation'].options);
 
-    const userInfo = sanitizeEntity(user, {
-      model: strapi.getModel('plugin::users-permissions.user'),
-    });
-
     const confirmationToken = crypto.randomBytes(20).toString('hex');
 
     await this.edit({ id: user.id }, { confirmationToken });
 
     settings.message = await userPermissionService.template(settings.message, {
       URL: `${getAbsoluteServerUrl(strapi.config)}/auth/email-confirmation`,
-      USER: userInfo,
+      USER: user,
```

Comment:
This function is very sensible to attacks. it should keep the sanitize in it. to avoid any wrong usage and security issues if missused

Reply:
Good catch, thanks for noticing!